src/jobs/codeql.yml

```diff
@@ -43,14 +47,25 @@ steps:
           cmd+=(--command '<< parameters.build-command >>')
```

Comment:
im going to assume that you intend to rather scan the binary from workspace (and maybe the artifacts from m2 etc) than actually rebuilding things. right?